README.md

```diff
@@ -3,30 +3,34 @@
 ![logo](images/logo.png "Logo")
 
 ## Table of Contents:
+- [Awesome Appwrite ](#awesome-appwrite-)
+  - [Table of Contents:](#table-of-contents)
   - [Announcements and Official Blog Posts](#announcements-and-official-blog-posts)
-    - [30 Days of Appwrite](#30-days-of-appwrite-)
-    - [Almost Netflix](#almost-netflix-)
+    - [30 Days of Appwrite ✨](#30-days-of-appwrite-)
+    - [Almost Netflix 🎬](#almost-netflix-)
   - [Hacktoberfest 2021](#hacktoberfest-2021)
+  - [Hacktoberfest 2022](#hacktoberfest-2022)
   - [Learning Resources](#learning-resources)
     - [Getting Started](#getting-started)
     - [Setup, Customization and Deployment](#setup-customization-and-deployment)
     - [Web Development](#web-development)
     - [Mobile Development](#mobile-development)
     - [Security and Authentication](#security-and-authentication)
     - [Appwrite Services](#appwrite-services)
-  - [Showcase (Source Code)](#showcase-built-with-appwrite-)
+  - [Showcase (Built with Appwrite 📣)](#showcase-built-with-appwrite-)
     - [Angular](#angular)
     - [Cordova](#cordova)
     - [Express](#express)
     - [Flutter](#flutter)
     - [Godot](#godot)
     - [Golang](#golang)
+    - [Preact](#preact)
     - [Python](#python)
     - [React](#react)
+    - [React Native](#react-native)
     - [Svelte](#svelte)
-    - [Unity](#unity)
     - [Vue](#vue)
-    - [Nuxt](#nuxt)
+    - [Unity](#unity)
```

Comment:
@apoorvdwi could you please revert the changes made to the Table of Contents?

Reply:
Done